modules/nvidia-gpu-aws-deploying-the-node-feature-discovery-operator.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+//  * installing/installing-nvidia-gpu-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-aws-deploying-the-node-feature-discovery-operator_{context}"]
+= Deploying the Node Feature Discovery Operator
+
+After the GPU-enabled node is created, you need to discover the GPU enabled node
+so it can be scheduled. To do this, install the Node Feature Discovery (NFD) Operator.
+
+The NFD operator identifies hardware device features in nodes.
+It solves the general problem of identifying and cataloging hardware resources in
+the infrastructure nodes so they can be made available to OpenShift.
+
+.Procedure
+
+. Install the Node Feature Discovery operator from **OperatorHub** in the
+**OpenShift Container Platform** console.
+
+. After installing the NFD operator into **OperatorHub**,
+select **Node Feature Discovery** from the installed operators list and
+select **Create instance**. This installs the ``openshift-nfd`` Operator
+into the ``openshift-operators`` namespace.
+
+. Verify the Operator is installed and running:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-operators
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME READY STATUS RESTARTS AGE
+
+nfd-operator-fd55688bd-4rrkq 1/1 Running 0 18m
+----
+
+. Browse to the installed operator in the console and select
+**Create Node Feature Discovery**.
+
+. Select **Create** to build a NFD Custom Resource. This creates NFD pods in
+the ``openshift-nfd`` namespace that poll the OpenShift nodes for hardware
+resources and catalogue them.
+
+. After a successful build, verify that a NFD pod is running on each nodes:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-nfd
+----
++
+.Example output
+[source,terminal]
+----
+NAME READY STATUS RESTARTS AGE
+
+nfd-master-mc99f 1/1 Running 0 51s
+
+nfd-master-t7rrl 1/1 Running 0 51s
+
+nfd-master-w9pgx 1/1 Running 0 51s
+
```

Comment:
Starting in OCP 4.12, there will be one nfd-master deployment pod per cluster, instead of one for each master node.

Reply:
@wabouhamad Could you provide a sample?

Reply:
@StephenJamesSmith : here's an example from OCP 4.12
`# oc get pods -n openshift-nfd
NAME                                                              READY   STATUS      RESTARTS        AGE
nfd-controller-manager-8646fcbb65-x5qgk                           2/2     Running     7 (8h ago)      12d
nfd-master-769656c4cb-w9vrv                                       1/1     Running     0               12d
nfd-worker-qjxb2                                                  1/1     Running     3 (3d14h ago)   12d
nfd-worker-xtz9b                                                  1/1     Running     5 (3d14h ago)   12d`

Reply:
Ok. Thanks. Made this change.